hack/update-vendor-licenses.sh

```diff
@@ -150,21 +150,18 @@ fi
 LICENSE_ROOT="${LICENSE_ROOT:-${KUBE_ROOT}}"
 cd "${LICENSE_ROOT}"
 
-VENDOR_LICENSE_FILE="Godeps/LICENSES"
-TMP_LICENSE_FILE="/tmp/Godeps.LICENSES.$$"
+TMP_LICENSE_FILE="/tmp/vendor.LICENSES.$$"
```

Comment:
The current script here is old, and needs to be brought up to current standards.

Around L32, we should add:
```sh
KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
source "${KUBE_ROOT}/hack/lib/init.sh"
```

Then we can use `kube::util::ensure-temp-dir` to create and ensure we have a temp dir that will be cleaned up. Then anything temp you need to make can be under `${KUBE_TEMP}/` (both the license file and the temp dir)